common/Makefile.am.inc

```diff
@@ -280,6 +280,7 @@ sources = $(libflatpak_common_la_SOURCES) $(libflatpak_la_SOURCES)
 
 INTROSPECTION_GIRS =
 
+if !ENABLE_ASAN
```

Comment:
Just a comment: I don't love silently disabling a feature, I'd rather see it error that options are incompatible or at least print a warning.

Reply:
agreed. I added a warning message.